src/crab/cfg.hpp

```diff
@@ -619,8 +619,13 @@ std::vector<std::string> stats_headers();
 
 std::map<std::string, int> collect_stats(const cfg_t&);
 
-cfg_t prepare_cfg(const InstructionSeq& prog, const program_info& info, bool simplify, bool check_for_termination,
-                  bool must_have_exit = true);
+struct prepare_cfg_options {
```

Comment:
Does it make sense to use this struct as a field in the ebpf_verifier_options_t?

Reply:
Good idea. Let me see if I can make this change.